.circleci/config.yml

```diff
@@ -66,6 +66,7 @@ commands:
       - docker/install-docker
       - go/install
       - kube-orb/install-kubectl
+      - run: go get -u github.com/tsenart/vegeta
```

Comment:
Are you getting the test dependency explicitly here to avoid
adding dependencies to go.mod?
If yes, can't we use a generic instruction like:
```go get -t -u ./test/...```

Reply:
Actually this is a mistake, should I add the dependency in the go.mod?, also the spew is a new dependency